### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by 202-ecommerce**

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" ?>
         <module>
             <name>reforestaction</name>
-            <displayName><![CDATA[Purchase responsable]]></displayName>
-            <version><![CDATA[1.0.1]]></version>
-            <description><![CDATA[With Reforest&#039;action, your customers plant a tree in the world to offset the CO2 emissions of their purchases. They do this for the planet and allow you to increase your visibility on social networks.]]></description>
+            <displayName><![CDATA[Achat responsable]]></displayName>
+            <version><![CDATA[1.0.2]]></version>
+            <description><![CDATA[Avec Reforest&#039;action, vos clients plantent un arbre dans le monde pour compenser les &eacute;missions de CO2 de leurs achats. Ils agissent ainsi pour la plan&egrave;te et vous gagnez en visibilit&eacute; sur les r&eacute;seaux sociaux. ]]></description>
```
Comment (Quetzacoalt91) [inline config.xml]: You have the wrong version written here

Comment (202-ecommerce) [inline config.xml]: Sorry,

I fix now.

File: config.xml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
         <module>
             <name>reforestaction</name>
-            <displayName><![CDATA[Purchase responsable]]></displayName>
-            <version><![CDATA[1.0.1]]></version>
+            <displayName><![CDATA[Achat responsable]]></displayName>
```
Comment (Quetzacoalt91) [inline config.xml]: Almost :D

Comment (202-ecommerce) [inline config.xml]: Oh ;(

It's ok now ? :dancer:

Comment (Quetzacoalt91) [inline config.xml]: Yes it is, I can now merge your PR.

Comment (202-ecommerce) [inline config.xml]: Ok thanks :)

**Comment by Quetzacoalt91** [comment]
Hi @202-ecommerce,

Thanks, your PR has just been merged and the changes should be available from tomorrow.

Best regards

**Pull request merged**